.github/ISSUE_TEMPLATE/blank_issue.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Blank issue"
+about: "Don’t see your issue here? Open a blank issue."
+title: ""
+labels: "triage label"
```

Comment:
I've added the label "needs triage", so I think we just need to update this line only?